apps/src/pythonlab/pythonHelpers/pythonScriptUtils.ts

```diff
@@ -210,6 +210,21 @@ export async function importPackagesFromFiles(
   }
 }
 
+// Delete any python globals that were not present in
+// the original globals on pyodide start (originalGlobals param).
+// This allows us to ensure each pyodide run has a clean state.
+export function resetGlobals(
+  pyodide: PyodideInterface,
+  originalGlobals: Map<string, string>
+) {
+  const newGlobals = pyodide.globals.toJs();
+  newGlobals.forEach((_value: string, key: string) => {
```

Comment:
I've always wished JS hadn't tried to make Map.forEach "compatible" with Array.forEach.... the reversed value/key always makes me gulp 🤣